modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/loader/java/AbstractJavaExtensionDeclarationTestCase.java

```diff
@@ -67,6 +68,13 @@ protected OperationDeclaration getOperation(WithOperationsDeclaration declaratio
                                        object -> ((OperationDeclaration) object).getName().equals(operationName));
   }
 
+  protected ConnectionProviderDeclaration getConnectionProvider(ConfigurationDeclaration configurationDeclaration,
```

Comment:
rename to `findConnectionProvider` and move to `org.mule.test.module.extension.internal.util.ExtensionsTestUtils` so that it's reusable by other tests